modules/network-observability-loki-install.adoc

```diff
@@ -5,7 +5,7 @@
 :_content-type: PROCEDURE
 [id="network-observability-loki-installation_{context}"]
 = Installing the Loki Operator
-It is recommended to install link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7], This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
+It is recommended to install link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]. This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
```

Comment:
"It" is a vague referent and recommending things to customers might leave us open to legal issues. Maybe structuring the opening sentence to something like the following would be a clearer solution?

> Consider installing Loki Operator version 5.7 so that you can use the using the `openshift-network` tenant configuration mode to create a LokiStack instance. Additonally, this version provides fully auotmatic, in-cluster uthentication and authorization support for Network Observability.

I see LokiStack in bold on https://docs.openshift.com/container-platform/4.13/logging/cluster-logging-loki.html.

Reply:
Should an attribute be created for the Loki Operator version? Eventually, 5.7 could become 5.8?

Reply:
@dfitzmau For better or worse, _It is recommended to _ is a very common phrase in the OCP docs (103 instances, according to Atom). It might be a discussion for a global change.

Reply:
Correct. I'd prefer to avoid it, but I'll leave my comment up to a writer as the style guides provide "guidelines" that are sometimes open to sytlistic preferences.

Reply:
A few good pts here! I'll respond below:

Comment 1: Point taken, "It" is vague and I also take the below point about the distinction between recommendation/consideration. I guess LokiStack refers to both a logging subsystem and also there is a LokiStack resource, which is referenced here in step 3. So this context, I'm specifically referencing the resource and you're correct that it needs some distinction in that regard, so I should put it in backticks.

Reply:
Should the word "must" be used for better enforcement of the need for Loki in the context of Network Observability?